models/_sector/dex/trades/bnb/_schema.yml

```diff
@@ -112,3 +112,62 @@ models:
             - token_sold_address
             - token_bought_amount_raw
             - token_sold_amount_raw
+
+  - name: pancakeswap_v2_bnb_base_trades
+    meta:
+      blockchain: bnb
+      sector: dex
+      project: pancakeswap
+      contributors: hosuke, chef_seaweed, tomfutago
+    config:
+      tags: [ 'bnb', 'dex', 'trades', 'pancakeswap', 'uniswap', 'v2' ]
+    description: "Pancakeswap BNB v2 base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_seed:
+          seed_file: ref('pancakeswap_bnb_base_trades_seed')
+          filter:
+            version:
+              - '2'
+              - 'mmpool'
+              - 'stableswap'
```

Comment:
This will now compile into `AND seed.version IN ('2', 'mmpool', 'stableswap')`.

Reply:
Nice! 💪 thanks for fixing this!